properties/P000100.md

```diff
@@ -3,7 +3,7 @@ uid: P000100
 slug: kc
 name: KC
 refs:
-  - doi: 10.1016/j.topol.2009.07.010
-    name: On some questions about KC and related spaces
+  - doi: 10.2307/2316017
```

Comment:
The previous reference was a more advanced article.  Replaced it with a more basic article, the Monthly article by Wilansky that introduced the KC terminology.